profiles/base.yaml

```diff
@@ -1452,9 +1452,13 @@ tarantula_debug: false
 # Number of minutes to wait between performing a verb on an item.
 faux_atmo_interval: 15
 
+# Set to true to randomly select an item and verb to perform.
+# Set to false to round-robin select items and verbs to perform.
+faux_atmo_random: true
+
 # Items that don't have an inherent atmo effect
 # but whose verbs if done periodically would.
-# faux_atmo_items:
+faux_atmo_items: []
```

Comment:
I would not initialize this to an empty array. `base-empty` will take care of that. You can just initialize it to nil (e.g. `faux_atmo_items:`) to make the base case simpler for new users.

Reply:
Ah, gotcha. I'll try that out

Reply:
Initialized `faux_atmo_items:` to nil in https://github.com/rpherbig/dr-scripts/pull/4492/commits/6761f49278069403637aac8fa007b9f63f65f2ae